tokio/src/io/mod.rs

```diff
@@ -231,6 +231,9 @@ cfg_io_driver_impl! {
     pub(crate) use poll_evented::PollEvented;
 }
 
+// The bsd module can't be build on Windows, so we completely ignore it, even
+// when building documentation.
+#[cfg(unix)]
```

Comment:
I wonder if we should abstract this into a macro that better describes what we are doing? Something like `cfg_ignore_in_windows_docs!`?